diff --git practica01/obj/b__main.ads practica01/obj/b__main.ads
index a58e968..889c210 100644
--- practica01/obj/b__main.ads
+++ practica01/obj/b__main.ads
@@ -40,7 +40,7 @@ package ada_main is
    pragma Export (C, main, "main");

    type Version_32 is mod 2 ** 32;
-   u00001 : constant Version_32 := 16#9ff6b36a#;
+   u00001 : constant Version_32 := 16#039d9ca2#;
    pragma Export (C, u00001, "mainB");
    u00002 : constant Version_32 := 16#2e11c0b1#;
    pragma Export (C, u00002, "system__standard_libraryB");
